governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx

```diff
@@ -13,26 +14,28 @@ import { StatusFilterProvider } from '../contexts/StatusFilterContext'
 import { classNames } from '../utils/classNames'
 import '../mappings/signers.json'
 
+const readPublisherKeyToNameMapping = (filename: string) => {
+  if (fs.existsSync(filename)) {
+    return YAML.parse(fs.readFileSync(filename, 'utf8')).reduce(
```

Comment:
In general there's not much reason to use `reduce`, it's much harder to grok for anyone who doesn't have a good FP background, and there are clearer ways to do nearly everything it does, see https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-reduce.md.

In this case I'd probably do this as:

```ts
return Object.fromEntries(arr.map(({ key, name }) => [key, name]))
```

Reply:
OK, the last time I used TS it was on a team with a functional background so it's where I will usually go. I'll change it.

Reply:
OK, this one is done.